File: Fallen Chimney/vaeltetskorsten.java
import java.util.Scanner;

public class vaeltetskorsten {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();

        int max = -1;

        for (int j = 0; j < n; j++) {
            int i = scanner.nextInt();
            String response = scanner.next();

            if (response.charAt(0) == 'n') {
                if (i > max) {
                    max = i;
                }
            }
        }

        System.out.println(max);
    }
}

File: Fallen Chimney/vaeltetskorsten.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int max = -1;

    for (int j = 0; j < n; j++) {
        int i;
        char response[4];
        scanf("%d %s", &i, response);

        if (response[0] == 'n') {
            if (i > max) {
                max = i;
            }
        }
    }
    
    printf("%d", max);
    return 0;
}
